src/Evenement/EventEmitterTrait.php

```diff
@@ -75,6 +75,12 @@ public function off($event, callable $listener = null)
         $this->removeAllListeners($event);
     }
 
+    public function eventNames() {
+        return \array_unique(
+            \array_merge(\array_keys($this->listeners), \array_keys($this->onceListeners), \array_keys($this->beforeOnceListeners))
+        );
+    }
+
```

Comment:
Looks interesting, but maybe add some documentation for it as well?

Reply:
I added documentation  to 01-api.md file.